test/e2e/operator_test.go

```diff
@@ -769,11 +876,25 @@ func TestGatewayOwnership(t *testing.T) {
 	testName := "test-gateway-owned"
 	contourName := fmt.Sprintf("%s-contour", testName)
 	gcName := "test-gatewayclass-owned"
+
+	// Configure the nodeports to match the kind config.
+	http := int32(30080)
+	httpPort := operatorv1alpha1.NodePort{
+		Name:       "http",
+		PortNumber: &http,
+	}
+	https := int32(30443)
+	httpsPort := operatorv1alpha1.NodePort{
+		Name:       "https",
+		PortNumber: &https,
+	}
+
 	cfg := objcontour.Config{
 		Name:         contourName,
 		Namespace:    operatorNs,
 		SpecNs:       specNs,
 		NetworkType:  operatorv1alpha1.NodePortServicePublishingType,
+		NodePorts:    []operatorv1alpha1.NodePort{httpsPort, httpPort},
```

Comment:
bit of a shame we have to repeat this, maybe add a helper function or variable for this if it is needed everywhere